startup/server/export.js

```diff
@@ -465,19 +467,21 @@ WebApp.connectHandlers.use("/admin/export", (req, res, next) => {
     playerInputDataFields
   );
 
-  const playerLogFields = [
-    "_id",
-    "playerId",
-    "gameId",
-    "roundId",
-    "stageId",
-    "name",
-    "jsonData",
-    "createdAt"
-  ];
-  saveFile("player-logs", playerLogFields, puts => {
-    batch(PlayerLogs)(p => puts(_.pick(p, playerLogFields)));
-  });
+  if (req.query.remove_pii === "false") {
+    const playerLogFields = [
+      "_id",
+      "playerId",
+      "gameId",
+      "roundId",
+      "stageId",
+      "name",
+      "jsonData",
+      "createdAt"
+    ];
+    saveFile("player-logs", playerLogFields, puts => {
+      batch(PlayerLogs)(p => puts(_.pick(p, playerLogFields)));
+    });
+  }
```

Comment:
Player logs should not be filtered out. They are game data, not PII (unless the experimenter did something wrong, which is beyond our reach for now) 👍